prow/cmd/deck/job_history.go

```diff
@@ -301,7 +339,7 @@ func getBuildData(bucket storageBucket, dir string) (buildData, error) {
 		b.commitHash = commitHash
 	}
 	finished := gcs.Finished{}
-	err = readJSON(bucket, path.Join(dir, "finished.json"), &finished)
+	err = readJSON(ctx, bucket, path.Join(dir, "finished.json"), &finished)
```

Comment:
Obviously not your PR but this filename really belongs into a `const` in an api package

Reply:
Yup. Added it to the list :)
https://github.com/kubernetes/test-infra/pull/17183#issue-402002610